File: CPP/spoj/prawie_jak_fibonacci_prawie.cpp
#include <iostream>
#include <vector>
using namespace std;

int main()
{
    int n, temp, ile = 0;
    vector<int> wejscie;

    cin >> n;
    for (int i = 0; i < n; i++)
    {
        cin >> temp;
        wejscie.push_back(temp);
    }

    for(int i = 0; i < (n-2); i++)
    {
        if (( wejscie[i+2] ) == (( wejscie[i+1] ) + ( wejscie[i] )))
        {
            ile += 1;
        }

    }
    cout << ile;
    return 0;
}

File: PY/spoj/prawie_jak_fibonacci_prawie.py
n = int(input())
ile = 0

wejscie = input().split()
for i in range(n-2):
    if int(wejscie[i+2]) == (int(wejscie[i+1]) + int(wejscie[i])):
        ile += 1

print(ile)
quit()